deafrica/monitoring/latency_check.py

```diff
@@ -111,7 +111,10 @@ def latency_checker(
             ds = dc.find_datasets(product=satellite, **query)
             print("Datasets since ", date_n_days_ago, " : ", len(ds))
 
-            s3_latency =s3_latency_check(Bucket="deafrica-landsat", Prefix="collection02/level-2/standard/etm/2023")
+            s3_latency = s3_latency_check(
+                Bucket="deafrica-landsat",

```

Comment:
You should not hard-code these inputs, they will overwrite the actual workflow inputs. Here you must call the function by passing the local variables (which you need to add, see other comment later) `Bucket` and `Prefix`. These are not default values.